components/popover/test/popover-mixin.test.js

```diff
@@ -3,16 +3,6 @@ import { aTimeout, expect, fixture, focusElem, html, nextFrame, oneEvent, runCon
 
 describe('popover-mixin', () => {
 
-	const open = (method, popover, applyFocus) => {
```

Comment:
These aren't needed. Not sure why I didn't do `elem[method]()` from the beginning.